src/clientsettingsdlg.cpp

```diff
@@ -201,7 +201,9 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                      "In this case the buffer delay setting is disabled and has to be "
                                      "changed using the sound card driver. On Windows, use the "
                                      "ASIO Device Settings button to open the driver settings panel. On Linux, "
```

Comment:
On Linux --> If you use JACK (e.g. on Linux)
since it's probably the same for the JACK on Windows version.

Reply:
I did not check the code now, but believe to remember this text is not showing on Windows with JACK. At least not currently, but might be good to be prepared for possible changes in the future.

Reply:
Could be. Thanks for the information. But, I'd say it's still worth changing.

Reply:
Yes, saying "On Windows" and then _only_ refering to ASIO is wrong.  So it needs changing.  Also, JACK also is not Linux-only.

These would be better phrased as
> Use the appropriate tool for the interface in use to adjust this buffer size.  For example, if using ASIO, use the "ASIO Device Settings" button to open the driver settings panel or if using JACK, use a tool such as qJackCtl to adjust the buffer size.  Other interfaces, such as Pipewire, would require their appropriate tool being used.  Please refer to the interface manual.